App crashes on `cmpChatsAndCats`

# Bug Report

## Description

<!-- Provide a short description describing the problem you are experiencing. -->

```
Traceback (most recent call last)
/home/osmaczny/Dev/status/status-desktop/src/nim_status_client.nim(199) nim_status_client
/home/osmaczny/Dev/status/status-desktop/src/nim_status_client.nim(196) mainProc
/home/osmaczny/Dev/status/status-desktop/vendor/nimqml/src/nimqml/private/qguiapplication.nim(29) exec
/home/osmaczny/Dev/status/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(49) qobjectCallback
/home/osmaczny/Dev/status/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(34) onSlotCalled
/home/osmaczny/Dev/status/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/core/macros.nim(559) onSlotCalled
/home/osmaczny/Dev/status/status-desktop/src/app_service/service/chat/service.nim(257) onAsyncGetChatsByChannelGroupIdResponse
/home/osmaczny/Dev/status/status-desktop/src/app/core/eventemitter.nim(58) emit
/home/osmaczny/Dev/status/status-desktop/src/app/modules/main/chat_section/controller.nim(115) :anonymous
/home/osmaczny/Dev/status/status-desktop/src/app/modules/main/chat_section/io_interface.nim(40) onChatsLoaded
/home/osmaczny/Dev/status/status-desktop/src/app/modules/main/chat_section/module.nim(389) onChatsLoaded
/home/osmaczny/Dev/status/status-desktop/src/app/modules/main/chat_section/module.nim(246) buildChatSectionUI
/home/osmaczny/Dev/status/status-desktop/src/app/modules/main/chat_section/model.nim(196) setData
/home/osmaczny/Dev/status/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/algorithm.nim(413) sort
/home/osmaczny/Dev/status/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/algorithm.nim(331) mergeAlt
/home/osmaczny/Dev/status/status-desktop/src/app/modules/main/chat_section/model.nim(189) cmpChatsAndCats
/home/osmaczny/Dev/status/status-desktop/src/app/modules/main/chat_section/item.nim(240) categoryPosition
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [Makefile:747: run-linux] Segmentation fault (core dumped)
```

## Steps to reproduce

I don't know the exact steps. I used a shared dev data folder. The app crashes quite frequently when loading channels for the first time, particularly when switching between sections. Out of several crashes, I received a meaningful stack trace only once.

- Status desktop version: master (70e8370aae07e5d6a0e2e08197db75d9ca53ec01)
- Operating System: Ubuntu

--- a/src/app/modules/main/module.nim
+++ b/src/app/modules/main/module.nim
@@ -750,6 +750,8 @@ proc notifySubModulesAboutChange[T](self: Module[T], sectionId: string) =
   # If there is a need other section may be notified the same way from here...
 
 method activeSectionSet*[T](self: Module[T], sectionId: string) =
+  if self.view.activeSection.getId() == sectionId:
+    return
   let item = self.view.model().getItemById(sectionId)
 
   if(item.isEmpty()):

